### PR TITLE
Added a button to reset colormap limits to data

### PR DESCRIPTION
**Pull request by loichuder**
image

### CODE REVIEW

**Comment by axelboc** [comment]
How about putting the button on the left of the slider and using an icon instead of text, like [`FiRotateCcw` or `FiRefreshCcw`](https://react-icons.netlify.com/#/icons/fi)?

File: src/h5web/visualizations/heatmap/DomainSlider.tsx [hunk 1]
```diff
@@ -33,25 +33,34 @@ function DomainSlider(): JSX.Element {
...
+          const roundedDomain = (values as number[]).map(value =>
+            round(value, MIN_DECIMALS)
+          ) as Domain;
+          setCustomDomain(roundedDomain);
+        }}
+      />
+      <button
+        className={styles.resetButton}
+        type="button"
+        onClick={() => setCustomDomain(dataDomain)}
```
Comment (axelboc) [inline src/h5web/visualizations/heatmap/DomainSlider.tsx:59]: You could disable the button if `customDomain` is `undefined`: `disabled={!customDomain}`

Comment (axelboc) [reply src/h5web/visualizations/heatmap/DomainSlider.tsx:59]: Oh and it should be:

```jsx
onClick={() => setCustomDomain(undefined)}
```

When the custom domain is `undefined`, the visualization falls back to using the `dataDomain`. It allows doing stuff if no custom domain is set (like disabling this button).

Comment (loichuder) [reply src/h5web/visualizations/heatmap/DomainSlider.tsx:59]: Clever :100:, I will do that !

**Pull request merged**